paccmann_predictor/models/bimodal_mca.py

```diff
@@ -79,52 +78,65 @@ def __init__(self, params, *args, **kwargs):
         self.params = params
         self.smiles_padding_length = params['smiles_padding_length']
         self.protein_padding_length = params['protein_padding_length']
-
+        self.encoding = params['encoding']
         self.loss_fn = LOSS_FN_FACTORY[
             params.get('loss_fn', 'binary_cross_entropy')
         ]  # yapf: disable
-
+        self.peptide_embedding = params['peptide_embedding']
```

Comment:
same here, also we should call it protein and not peptide to be consistent with the remainder of the code